content/en/getting_started/synthetics/browser_test.md

```diff
@@ -19,77 +22,117 @@ further_reading:
-    - **Browsers & Devices**: Choose the devices and browsers you want to use for testing. In this example, the test is only run from **Chrome** and on **Large Laptops**.
-    - **Locations**: Choose one of the **Managed Locations** to run your test from. In this example, the test is run in **Americas and Europe**.
-    - **Specify a test frequency**: Select how often you would like the test to run.
-    - **Alert Conditions**: Set alert conditions to determine the circumstances under which you want a test to send a notification alert.
+You can specify the number of failures per browser and device before the test is considered failed:
 
-        - To avoid being alerted on network blips that might happen on specific locations, this test is configured as:
+```text
+Retry 1 time after 300 ms in case of failure
+```
 
-        ```text
-        An alert is triggered if your test fails for 0 minutes from any 3 of 13 locations
-        ```
+**Note**: By default, the wait time for a failed test to retry is 300ms. You can configure this interval with the [Synthetics API][3].
 
-        - To ensure that a test execution is only considered a failure after two failed test runs, specify how many times you want your test to be re-run before being considered a failure:
+To avoid being alerted on network blips that might happen on specific locations, set up an alerting condition such as:
 
-        ```text
-        Retry 1 time before location is marked as failed
-        ```
+```text
+An alert is triggered if your test fails for 0 minutes from any 3 of 13 locations
+```
 
-         **Note**: By default, there is a 300ms wait before retrying a test that failed. This interval can be configured via the [API][3].
+#### Notify your team
 
-    - **Notify**: Write an alert message and specify which email addresses should be notified when the alert is triggered. No additional set up is required to start receiving alert emails from Datadog. You can also use [integrations][4], such as Slack, PagerDuty, webhooks, etc., to receive alert notifications.
-    - Click **Save & Edit Recording**.
+Add an alert name to the **Monitor Name** field and write a message for the alert. You can tag other teams, specify which service and team members receive the alert notifications, and use [integrations][4] such as Slack, PagerDuty, and webhooks.
+
+You can set time for your alert notification to re-notify if the alert has not been resolved and define the priority of the alert, ranging from **P5 (Info)** to **P1 (Critical)**.
+
+When you're ready to run your test, click **Save Details & Record Test**.
 
 {{< img src="getting_started/synthetics/configured-browser-test.gif" alt="Configured browser test"  style="width:90%;">}}
 
-## Record your test steps
+### Create recording
+
+Once your test configuration has saved, Datadog prompts you to download and install the [Datadog test recorder][5] extension. You can run browser tests on Google Chrome, Microsoft Edge, and Firefox. 
```

Comment:
```suggestion
Once your test configuration has saved, Datadog prompts you to download and install the [Datadog test recorder][5] extension. Browser tests are able to run on Google Chrome, Microsoft Edge, and Firefox. 
```

Reply:
Do we have links to Microsoft Edge / Firefox? This link is for the Chrome extension.

Reply:
And I oop! Good catch on the link, I'll change it to Google Chrome (https://www.google.com/chrome/).

@sarina-dd is there a way to remove the EN-US part of these page URLs?

https://www.mozilla.org/en-US/firefox/new/
https://www.microsoft.com/en-us/edge